scvi/dataset/dataset.py

```diff
@@ -182,7 +182,7 @@ def _cell_type_idx(self, cell_types):
 
     def _gene_idx(self, genes):
         if type(genes[0]) is not int:
-            genes_idx = [np.where(gene == self.gene_names)[0][0] for gene in genes]
+            genes_idx = [np.where(gene.lower() == self.gene_names)[0][0] for gene in genes]
```

Comment:
It matters when you want to match the genes between the two datasets (you do string equality).
I had issues with one dataset being uppercase and one being a mix but mostly lowercase.

Once again, I guess we can do that in #374 because for now all datasets have different behaviors.

Reply:
You're right uppercase seems to be standard (even though it always hurt my eyes).